certora/funnels/automation/VaultMinter.spec

```diff
@@ -0,0 +1,327 @@
+// VaultMinter.spec
+
+methods {
+    function wards(address) external returns (uint256) envfree;
+    function buds(address) external returns (uint256) envfree;
+    function config() external returns (int64, uint32, uint32, uint128) envfree;
+    function vault() external returns (address) envfree;
+    function _.draw(uint256 wad) external => drawSummary(calledContract, wad) expect bool; // Forcing to have a return value
```

Comment:
What does forcing the return value accomplish? Seems like the actual `AllocatorVault` methods do not have return values.

Reply:
So this is the error if you do not force a return value: "Wildcard method entry with summary drawSummary(calledContract,wad) must include an expected return type".
I pushed a better comment about this.